inc/columns.class.php

```diff
@@ -1,120 +1,105 @@
 <?php
-// Make sure this file is only run from within the WordPress context.
-defined( 'ABSPATH' ) || exit;
-
+/**
+ * The Columns Class.
+ *
+ * @package Antispam Bee
+ */
 
-// Antispam_Bee_Columns
+defined( 'ABSPATH' ) || exit;
 
+/**
+ * Class Antispam_Bee_Columns
+ */
 final class Antispam_Bee_Columns {
 
-
 	/**
-	* Register plugin columns on comments screen
-	*
-	* @since   2.6.0
-	* @change  2.6.0
-	*
-	* @param   array  $columns  Array with existing columns
-	* @return  array            Array with extended columns
-	*/
-
-	public static function register_plugin_columns($columns)
-	{
+	 * Register plugin columns on comments screen.
+	 *
+	 * @since   2.6.0
+	 * @change  2.6.0
+	 *
+	 * @param   array $columns Array with existing columns.
+	 * @return  array          Array with extended columns.
+	 */
+	public static function register_plugin_columns( $columns ) {
 		return array_merge(
 			$columns,
 			array(
-				'antispam_bee_reason' => esc_html__('Spam Reason', 'antispam-bee')
+				'antispam_bee_reason' => esc_html__( 'Spam Reason', 'antispam-bee' ),
 			)
 		);
 	}
 
-
 	/**
-	* Display plugin column values on comments screen
-	*
-	* @since   2.6.0
-	* @change  2.6.0
-	*
-	* @param   string   $column      Currently selected column
-	* @param   integer  $comment_id  Comment ID
-	*/
-
-	public static function print_plugin_column($column, $comment_id)
-	{
-		// Only Antispam Bee column
-		if ( $column !== 'antispam_bee_reason' ) {
+	 * Display plugin column values on comments screen
+	 *
+	 * @since   2.6.0
+	 * @change  2.6.0
+	 *
+	 * @param   string  $column      Currently selected column.
+	 * @param   integer $comment_id  Comment ID.
+	 */
+	public static function print_plugin_column( $column, $comment_id ) {
+		if ( 'antispam_bee_reason' !== $column ) {
 			return;
 		}
 
-		// Init data
-		$spam_reason = get_comment_meta($comment_id, $column, true);
+		$spam_reason  = get_comment_meta( $comment_id, $column, true );
 		$spam_reasons = Antispam_Bee::$defaults['reasons'];
```

Comment:
With or without whitespace in the brackets Line 48 vs. line 50; I mean WP codex need the space. @websupporter